CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
-# dbt_jira VERSION.VERSION
+# dbt_jira v0.16.0
+[PR #122](https://github.com/fivetran/dbt_jira/pull/122) contains the following updates:
+
+## 🚨 Breaking Changes: Bug Fixes 🚨
+- The following fields in the below mentioned models have been converted to a string datatype (previously integer) to ensure classic Jira projects may link issues to epics. If you are referencing these fields downstream, be sure to make any changes to account for the new datatype.
```

Comment:
It might help to add a line explaining how the classic projects were formatted, or why this benefits the classic projects.

Reply:
Yeah I think that would be helpful. Just added a sentence explaining this a bit more.

Reply:
Super helpful!